src/Config.php

```diff
@@ -565,7 +565,7 @@ public function showFormInventory()
 
         $fields = ["contact", "user", "group", "location"];
         echo "<tr class='tab_bg_2'>";
-        echo "<td> " . __('When connecting or updating') . "</td>";
+        echo "<td> " . __('When connecting or updating the relevant field') . "</td>";
```

Comment:
I guess this is enough:
```suggestion
        echo "<td> " . __('When connecting or updating the field') . "</td>";
```

(wait for other advices)

Reply:
I'm more inclined to insist ^^